app/test_main.py

```diff
@@ -1,3 +1,50 @@
+from typing import Any
+
 from app.main import get_human_age
+import pytest
```

Comment:
Add a blank line between different import groups and order them correctly to follow PEP-8